_opcodes/op67-bgee.html

```diff
@@ -0,0 +1,56 @@
+---
+n: 67
+name: "Summon: Creature Summoning"
+param1: "Unknown"
+param2: "Control"
+bg1: 0
+bg2: 0
+bgee: 1
+iwd1: 0
+iwd2: 0
+pst: 0
+---
+Summons a creature, with EA set to the 'Control' field. The Resource key should be set to the filename of the creature to summon.
+<br/>Known values for 'Control' are:
+<ul>
+	<li>0 &longrightarrow; Match target (hostile if no target)</li>
+	<li>1 &longrightarrow; Match target (hostile if no target)</li>
+	<li>2 &longrightarrow; As creature file</li>
+	<li>3 &longrightarrow; Match target (hostile if no target)</li>
+	<li>4 &longrightarrow; As creature file</li>
+	<li>5 &longrightarrow; Hostile to target</li>
+	<li>6 &longrightarrow; As creature file</li>
+	<li>8 &longrightarrow; As creature file</li>
+	<li>255 &longrightarrow; As creature file</li>
+</ul>
+
+<br/><br/>If effect's target is set as 'Original Caster' then the 'Control' parameter covers 'Location' as well as control. In this case, known values for 'Control' are:
+<ul>
+	<li>2 &longrightarrow; Summoned at location of target creature</li>
+	<li>4 &longrightarrow; Summoned at location of caster</li>
+	<li>6 &longrightarrow; Summoned at location of caster</li>
+</ul>
```

Comment:
BTW, this looks completely redundant. I know it was picked up from the previous description, but if the caster is the target, then all these three point to the same location.

Reply:
So I should remove it...?

Reply:
Sure, it would improve things.

Reply:
Removed.